tests/Unit/DataStructures/Test_DenseVector.cpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include "DataStructures/DenseVector.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.DataStructures.DenseVector", "[DataStructures][Unit]") {
+  // Since `DenseVector` is just a thin wrapper around `blaze::DynamicVector` we
+  // test a few operations and refer to Blaze for more thorough tests.
+  DenseVector<double> vec(3UL, 1.);
```

Comment:
Notwithstanding the examples in their documentation, this constructor takes a `size_t`.  (And again below.)

Reply:
3UL -> 3